### ISSUE TITLE
False-positive in `unnecessary_lambda_linter`?

### ISSUE THREAD

**Issue by strengejacke** [r-lib/lintr#2247]
I got a warning from lintr:

> Warning: file=R/check_predictions.R,line=434,col=30,[unnecessary_lambda_linter] Pass exp directly as a symbol to lapply() instead of wrapping it in an unnecessary anonymous function. For example, prefer lapply(DF, sum) to lapply(DF, function(x) sum(x)).

The related code line is:

```r
sims[] <- lapply(sims, function(i) exp(i) - plus_minus)
```

This looks like a false-positive to me?

**Comment by MichaelChirico** [comment]
Thanks for the fix!

**Comment by strengejacke** [comment]
Thanks for fixing this so quickly!
